database/seeders/UserSeeder.php

```diff
@@ -4,15 +4,32 @@
 
 namespace Database\Seeders;
 
+use App\Models\School;
 use App\Models\User;
+use Carbon\Carbon;
 use Illuminate\Database\Seeder;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Str;
+
+use function fake;
 
 class UserSeeder extends Seeder
 {
     public function run(): void
     {
-        User::factory()->superAdmin()->create();
-        User::factory()->admin()->create();
-        User::factory()->create();
+        for ($i = 1; $i < 10; $i++) {
```

Comment:
Why you don't use `UserFactory` ? You can pass parameters to `UserFactory`.

Reply:
`User::factory()->count(10)->create()`

Reply:
fixed